src/reactThisBindingIssueRule.ts

```diff
@@ -54,6 +69,9 @@ class ReactThisBindingIssueRuleWalker extends ErrorTolerantWalker {
             if (typeof(opt) === 'object') {
                 this.allowAnonymousListeners = opt['allow-anonymous-listeners'] === true;
             }
+            if (typeof opt === 'string') {
```

Comment:
IMHO, it would be better to add "speaking" property to object, instead of listing all passes strings as decorators. So setup will be like
```
"react-this-binding-issue": [true, {
    "allow-anonymous-listeners": true,
    "bind-decorators": ["autobind", "custom-bind"]
}]
```
instead of
```
"react-this-binding-issue": [true, "autobind", "customBind", { "allow-anonymous-listeners": true}]
```

But it's up to maintainers.

Reply:
This is something I didn't find much examples of or documentation, how do I write a schema? because options is of type `any`